src/main/fc/cli.c

```diff
@@ -598,7 +598,7 @@ static const clivalue_t valueTable[] = {
     { "max_throttle",               VAR_UINT16 | MASTER_VALUE, .config.minmax = { PWM_RANGE_ZERO, PWM_RANGE_MAX }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, maxthrottle) },
     { "min_command",                VAR_UINT16 | MASTER_VALUE, .config.minmax = { PWM_RANGE_ZERO, PWM_RANGE_MAX }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, mincommand) },
 #ifdef USE_DSHOT
-    { "digital_idle_percent",       VAR_FLOAT  | MASTER_VALUE, .config.minmax = { 0, 20 }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, digitalIdleOffsetPercent) },
+    { "digital_idle_percent",       VAR_FLOAT  | MASTER_VALUE, .config.minmax = { 0.0f, 20.0f }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, digitalIdleOffsetPercent) },
```

Comment:
Converting `minmax` to `float` will cause an explosion in the size of the CLI valuetable. Currently each `minmax` or `lookup` value uses 4 bytes, this change would make it 8 bytes, so we add 4 bytes per line in the value table. I guess there are at least 200 items in the value table, so this will cost about 800 bytes. That's significant even for F3 targets.

I suggest that where a non-int limit is required, it is checked in `validateAndFixConfig`.